pkg/query/query.go

```diff
@@ -298,45 +298,47 @@ func (s *Statement) Execute(ctx context.Context) (result *results.Result, err er
 		}
 	}
 
-	var rs = make(results.Rows, agg.aggregatedMap.Len())
+	var rs = make(results.Rows, agg.aggregatedMaps.Len())
 	count := 0
 
-	for i := agg.aggregatedMap.Iter(); i.Next(); {
+	for iface, aggMap := range agg.aggregatedMaps {
+		for i := aggMap.Iter(); i.Next(); {
 
-		key := types.ExtendedKey(i.Key())
-		val := i.Val()
+			key := types.ExtendedKey(i.Key())
```

Comment:
Editor whitespace hiccup?

Reply:
Mh, looks correct here locally (but I merged `develop`, so maybe it got fixed in that process)...